### ISSUE TITLE
catalog.register still not working in 0.6.0

### ISSUE THREAD

**Issue by rvrignaud** [gmr/consulate#59]
Hello,

With consulate 0.6.0 I still can't use catalog.register. See following stack trace: 

```
>>> import consulate
>>> c = consulate.Consul('192.168.59.103')
>>> c.catalog.datacenters()
u'dc1'
>>> c.catalog.register('test','192.168.0.1')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/rvrignaud/.virtualenvs/cr/lib/python2.7/site-packages/consulate/api/catalog.py", line 95, in register
    return self._adapter.put(['register'], payload)
  File "/Users/rvrignaud/.virtualenvs/cr/lib/python2.7/site-packages/consulate/adapters.py", line 42, in inner
    return fun(*args, **kwargs)
  File "/Users/rvrignaud/.virtualenvs/cr/lib/python2.7/site-packages/consulate/adapters.py", line 99, in put
    timeout=self.timeout))
  File "/Users/rvrignaud/.virtualenvs/cr/lib/python2.7/site-packages/requests/sessions.py", line 518, in put
    return self.request('PUT', url, data=data, **kwargs)
  File "/Users/rvrignaud/.virtualenvs/cr/lib/python2.7/site-packages/requests/sessions.py", line 451, in request
    prep = self.prepare_request(req)
  File "/Users/rvrignaud/.virtualenvs/cr/lib/python2.7/site-packages/requests/sessions.py", line 382, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "/Users/rvrignaud/.virtualenvs/cr/lib/python2.7/site-packages/requests/models.py", line 304, in prepare
    self.prepare_url(url, params)
  File "/Users/rvrignaud/.virtualenvs/cr/lib/python2.7/site-packages/requests/models.py", line 362, in prepare_url
    to_native_string(url, 'utf8')))
requests.exceptions.MissingSchema: Invalid URL "['register']": No schema supplied. Perhaps you meant http://['register']?
```

Thoughts ?

**Comment by gmr** [comment]
Fixed, will be available in 0.7 -- sorry for the delay.

**Comment by grobinson-blockchain** [comment]
Hello, are there any plans to release 0.7? I found this GitHub issue after debugging the same issue myself.
